InvenTree/InvenTree/urls.py

```diff
@@ -79,13 +80,16 @@
     # InvenTree information endpoint
     path('', InfoView.as_view(), name='api-inventree-info'),
 
-    # Third party API endpoints
-    path('auth/', include('dj_rest_auth.urls')),
-    path('auth/registration/', include('dj_rest_auth.registration.urls')),
-    path('auth/providers/', SocialProvierListView.as_view(), name='social_providers'),
-    path('auth/social/', include(social_auth_urlpatterns)),
-    path('auth/social/', SocialAccountListView.as_view(), name='social_account_list'),
-    path('auth/social/<int:pk>/disconnect/', SocialAccountDisconnectView.as_view(), name='social_account_disconnect'),
+    # Auth API endpoints
+    path('auth/', include([
+        re_path(r'^registration/account-confirm-email/(?P<key>[-:\w]+)/$', ConfirmEmailView.as_view(), name='account_confirm_email'),
+        path('registration/', include('dj_rest_auth.registration.urls')),
+        path('providers/', SocialProvierListView.as_view(), name='social_providers'),
+        path('social/', include(social_auth_urlpatterns)),
+        path('social/', SocialAccountListView.as_view(), name='social_account_list'),
+        path('social/<int:pk>/disconnect/', SocialAccountDisconnectView.as_view(), name='social_account_disconnect'),
+        path('', include('dj_rest_auth.urls')),
```

Comment:
This should be the first item so we can overwrite several defaults from dj-rest-auth

Reply:
@matmair sorry which line in particular should be first?